src/pages/Account/Profile.tsx

```diff
@@ -44,8 +45,10 @@ const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
   const onSubmit = async (data: ProfileFormData) => {
     if (!user) return;
     await updateUserProfile(user.uid, data);
-    const accountQuestionsUrl = `/account/questions?venueId=${venueId}&returnUrl=${window.location.pathname}${window.location.search}`;
-    const nextUrl = venueId ? accountQuestionsUrl : `/${DEFAULT_VENUE}`;
```

Comment:
These are the core of the change. `returnUrl` should never take the user back to part of the account creation flow; that's a recipe for loops.

And the `DEFAULT_VENUE` is going to confusingly send _someone_ to the wrong venue at some point, so it needs to go.